ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -0,0 +1,1560 @@
+{
```

Comment:
<p>What version of TF is being used in this tutorial? Do we still need the TF 2.3 note?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I think we can assume the user is using the latest version of TF, which we test for.</p>

Reply:
Done